### ISSUE TITLE
smells viz : y doesn't match any field name in datatable

### ISSUE THREAD

**Issue by mhow2** [crossminer/scava#218]
See example below. To me the name shouldn't be `Smells` but `ImproperShellSmells` or at least that the name of the columns match the `x`/`y` fields name.

It might affect others metrics.

```json
  {
    "id": "docker.smells.improperShellSmells",
    "name": "Improper Shell Docker Smells",
    "description": "The number of Improper Shell Docker smells per day, up to and including the current date.",
    "type": "LineChart",
    "datatable": {
      "cols": [
        {
          "name": "Date",
          "field": "$__date"
        },
        {
          "name": "Smells",
          "field": "$numberOfImproperShellSmells"
        }
      ]
    },
    "x": "Date",
    "y": "ImproperShellSmells",
    "timeSeries": true,
    "metricId": "org.eclipse.scava.metricprovider.historic.configuration.docker.smells"
  },
```

**Comment by valeriocos** [comment]
Thank you @mhow2 for reporting this issue. @blueoly please let me know once the bug is fixed because the scava2es code should be updated accordingly: https://github.com/crossminer/scava/blob/5466590d213a050b79312958e367163866f61380/web-dashboards/scava-metrics/scava2es.py#L225

thanks

**Comment by blueoly** [comment]
Thank you @mhow2 for your suggestions. I fixed the names and merged (714be60) the changes into dev branch.

**Comment by mhow2** [comment]
Look good to me, thank you @blueoly
